doc/GettingStartedWithTests.md

```diff
@@ -0,0 +1,91 @@
+---
+uid: Uno.Extensions.GettingStartedTests
```

Comment:
@jeromelaban l synced with @dansiegel and we think this documentation for tests should not be at the Uno.Extensions level in our opinion. What you be the best section in our opinion to have them? 
(GetStarted section? or is it a "too advanced" topic for the get started section?)

Reply:
Indeed, parts (2 to 5) that detail how to use unit test should be in unolatform/uno, then `Overview/Authoring`. The last part, 6 for server is not needed (it's not related to tests). @eriklimakc can you make a separate PR just for this topic? Thanks!

Reply:
@vatsashah45 has a PR for this https://github.com/unoplatform/uno/pull/16314